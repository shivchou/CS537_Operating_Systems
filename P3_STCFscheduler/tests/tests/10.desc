Testing multiple exec2() system call
