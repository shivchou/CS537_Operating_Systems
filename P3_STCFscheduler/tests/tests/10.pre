./edit-makefile.sh ../solution/Makefile test_10 > ../solution/Makefile.test
cp -f tests/test_10.c ../solution/test_10.c
cd ../solution/
make clean
make -f Makefile.test clean
make -f Makefile.test xv6.img
make -f Makefile.test fs.img
cd ../tests
