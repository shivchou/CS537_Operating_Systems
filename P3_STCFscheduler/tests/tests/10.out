XV6_TEST: scheduler : STCF
XV6_TEST: parent start
XV6_TEST: pid 8 started
XV6_TEST: pid 8 finished
XV6_TEST: pid 7 started
XV6_TEST: pid 7 finished
XV6_TEST: pid 4 started
XV6_TEST: pid 4 finished
XV6_TEST: pid 5 started
XV6_TEST: pid 5 finished
XV6_TEST: pid 6 started
XV6_TEST: pid 6 finished
